Add documentation for removing agent elevated privileges (#4910)
elastic/docs-content: reference/fleet/elastic-agent-unprivileged.md

## Changing an {{agent}}'s privilege mode [unprivileged-change-mode]

> For any installed {{agent}} you can change the mode that it's running in by running the `privileged` or `unprivileged` subcommand. Alternatively, you can [remove agents' elevated privileges centrally using the {{fleet}} UI or API](/reference/fleet/remove-agent-elevated-privileges.md), which is particularly useful when managing multiple agents.

Reviewer: Which method should be the main or preferred one? If the current order is correct, I said nothing. :) A suggestion:

```suggestion
For any installed {{agent}} you can change the mode that it's running in by running the `privileged` or `unprivileged` subcommand. 

:::{tip}
When managing multiple agents, [remove agents' elevated privileges centrally using the {{fleet}} UI or API](/reference/fleet/remove-agent-elevated-privileges.md).
:::
```
